sample/EasyTables/CSharp/QueueToInItemToOutItem/run.csx

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Microsoft.Azure.WebJobs.Host;
+using Newtonsoft.Json.Linq;
+
+public static void Run(string input, Item queriedItem, out Item newItem)
+{
+    newItem = new Item();    
+    newItem.Text = input;
+    queriedItem.Text += $"...{input}";
+}
+
+public class Item
```

Comment:
I agree. I was going to ask fabio about that today.

---
In reply to: [56189077](https://github.com/Azure/azure-webjobs-sdk-script/pull/98#discussion_r56189077) [](ancestors = 56189077)